VERSION

```diff
@@ -1 +1 @@
-0.8.16-dev
+0.8.16
```

Comment:
there's no point in changing the version number here on the `plugin-method` branch